### PR TITLE
Add entities and defer changes in Explorer

### PR DESCRIPTION
**Pull request by papollosc**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
-## 2023-02-11
+## 2023-02-14
 | What's new | |
 | :--------- | :-: |
-| **Add a Variant right from Studio**, via the new 'Add Variant' button in your Graph Settings. | ![image](https://user-images.githubusercontent.com/14367451/218590889-d8aaa9d8-f11f-4c54-8b2d-18c880798e8a.png)
+| With the right permission, **easily distinguish which types are entities in Explorer with an `E` icon!**| ![image](https://user-images.githubusercontent.com/86634858/218832452-469b58cf-4806-4c7c-a326-1be10f125a9a.png)
```
Comment (mayakoneval) [inline CHANGELOG.md:4]:
```suggestion
| **Easily distinguish which types are entities in Explorer with an `E` icon!**| ![image](https://user-images.githubusercontent.com/86634858/218832452-469b58cf-4806-4c7c-a326-1be10f125a9a.png)
```

Entities should be available for all roles

**Review by mayakoneval — APPROVED** [review]
LGTM, we still need to QA & flip the flag this week, last PR is waiting on review 🙏🏻

File: CHANGELOG.md (hunk 1 above)
Comment (papollosc) [reply CHANGELOG.md:4]: oh really! even consumers!! yay!

**Comment by papollosc** [comment]
> LGTM, we still need to QA & flip the flag this week, last PR is waiting on review 🙏🏻

Perfect! I have this ready to go for once ahead of time! :frog-laugh: Let me know when and we can merge this.

**Pull request merged**